src/docs/release/breaking-changes/eliminating-nullok-parameters.md

```diff
@@ -35,59 +35,60 @@ threw an exception when the sought-after widget was not present, and one that
 returned a nullable value that didn't throw an exception, and returned null if
 the widget was not present.
 
-The design document for this change is here:
-[https://flutter.dev/go/eliminating-nullok-parameters](
-https://flutter.dev/go/eliminating-nullok-parameters)
+The design document for this change is [Eliminating nullOk parameters][].
+
+[Eliminating nullOk parameters]: /go/eliminating-nullok-parameters
+
 
 ## Description of change
 
 The actual change modified these APIs to not have a `nullOk` parameter, and to
 return a non-nullable value:
 
-1. [`MediaQuery.of`]
```

Comment:
Seems weird to have a numbered list instead of a bulleted list, but maybe there's a reason?